lib/src/chart/base/axis_chart/scale_axis.dart

```diff
@@ -0,0 +1,20 @@
+enum ScaleAxis {
```

Comment:
Doesn't it make sense to use [PanAxis](https://api.flutter.dev/flutter/widgets/PanAxis.html) instead?
(I see that we have `PanAxis.aligned` which is probably challenging to implement here, but we can disable the functionality for now and maybe someday we'll be able to implement that)

In this way, we're consistent with the framework.

Reply:
I agree that it would be great to align with the Framework. 

`PanAxis` only applies to panning via touch gestures. It doesn't work with scrolling/trackpad and does not prevent scaling.

For `fl_chart` we want the developer to define the axis in which they want users to be able to scale the chart. Hence `ScaleAxis` should lead to more clarity and clearly differentiate from the differing behavior of `PanAxis`.

Reply:
Makes sense. So let's keep ScaleAxis
What do you think to add the `Fl` in front of it? (as we have this convention in the fl_chart's codebase)

Reply:
Following the convention makes sense. And it will also help prevent possible name collisions 👍 done ✅